### DISCUSSION TITLE
Getting content blocked for youtube videos

### DISCUSSION THREAD

**Discussion by MarkWasfy00** [thuliteio/doks#870] [Q&A] [answered]
#### i get content blocked after hosting my site on netlify 
![Screenshot 2022-08-16 084901](https://user-images.githubusercontent.com/76422492/184815607-2b381ef9-c5ff-4211-84f3-8a75bdefd6bd.png)

**Comment by h-enk** [comment, marked as answer]
Open Developer tools in your browser (F12) + fix the Content Security Policy issues you see in the Console tab.

See also the Doks docs: [Content Security Policy](https://getdoks.org/docs/reference/security/#content-security-policy)
